grakn-test/src/test/java/ai/grakn/test/engine/user/SuperUserTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Grakn - A Distributed Semantic Database
+ * Copyright (C) 2017  Grakn Labs Ltd
+ *
+ * Grakn is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Grakn is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
+ */
+package ai.grakn.test.engine.user;
+
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import ai.grakn.engine.GraknEngineConfig;
+import ai.grakn.engine.user.UsersHandler;
+import ai.grakn.test.EngineContext;
+import mjson.Json;
+
+/**
+ * Testing behavior of admin/super user.
+ * 
+ * @author borislav
+ *
+ */
+public class SuperUserTest {
+    
+    private static UsersHandler users = UsersHandler.getInstance();
+    
+    @ClassRule
+    public static final EngineContext engine = EngineContext.startInMemoryServer();
+    
+    @Test
+    public void testSuperuserPresent() {
+        Assert.assertNotNull(users.getUser(users.superUsername()));
+    }
+    
+    @Test
+    public void testSuperuserLogin() {        
```

Comment:
Nitpick: I think we should all be moving towards the test naming scheme where we write `whatTestDoes_expectedResult`.